src/routes/example.js

```diff
@@ -0,0 +1,26 @@
+import Boom from '@hapi/boom'
```

Comment:
Removed all reference to "Controller" as that feels more like we're claiming this is an MVC pattern when that's not the reality of Hapi.

Renamed `routes` to match Hapi terminology.  Majority of repos in Defra have a dedicated routes folder for these.  So adapting repo to fit most common use case.

Felt like the files were split too much, especially for a basic example, couldn't see much value in separating the handlers.

Reply:
Funnily enough I have renamed these index files to routes in some of our portal services. fEx Portal Frontend. I also have this on my TODO list to move to this in templates

The admittedly "over splitting" is a nod to how to split things when things grow.

- `controller.js` for 1 thing
- `controllers` folder for more than 1 thing

Not sure if we want to keep this?